src/lib.rs

```diff
@@ -400,6 +400,18 @@ impl<K, V, S> OrderMap<K, V, S>
     pub fn capacity(&self) -> usize {
         usable_capacity(self.raw_capacity())
     }
+
+    /// Clears the `OrderMap`, returning all key-value pairs as a `Drain`.
```

Comment:
Iterators don't really have a type-identity, they are peculiar that way. The struct is just a vehicle for the implementation, so we don't call it a `Drain` but maybe a drain iterator.

I think that the method should move down the method list to be placed at a location that degrades it in importance and where users more expect it... after the other iterator methods.